nimare/transforms.py

```diff
@@ -179,10 +184,163 @@ def resolve_transforms(target, available_data, masker):
             return None
         varcope = masker.inverse_transform(varcope.squeeze())
         return varcope
+    elif target == "p":
+        if ("t" in available_data.keys()) and ("sample_sizes" in available_data.keys()):
+            dof = sample_sizes_to_dof(available_data["sample_sizes"])
+            t = masker.transform(available_data["t"])
+            z = t_to_z(t, dof)
+            p = z_to_p(z)
+        elif "z" in available_data.keys():
+            z = masker.transform(available_data["z"])
+            p = z_to_p(z)
+        else:
+            return None
+        p = masker.inverse_transform(p.squeeze())
+        return p
     else:
         return None
 
 
+class CoordinateGenerator(Transformer):
+    """Extract peak statistical coordinates from statistical z or p maps.
+
+    Parameters
+    ----------
+    merge_strategy : :obj:`str`, choice of 'fill', 'replace', or 'demolish'
```

Comment:
```suggestion
    merge_strategy : {'fill', 'replace', 'demolish'}, optional
```

Pretty sure this is the preferred numpydoc approach.